upgrade.adoc

```diff
@@ -0,0 +1,494 @@
++
+[CAUTION]
+====
+If the original install was done with Ubuntu 12.04.2 or earlier,
+manual intervention is required to switch to the saucy (3.11) or later
+kernel series. See https://wiki.ubuntu.com/Kernel/LTSEnablementStack
+for further information.
+====
++
+
+You can find your current kernel version by running:
+
+----
+uname -a
+----
+
+For example:
+
+----
+build@zre-ubuntu12-64:~$ uname -a
+Linux zre-ubuntu12-64 3.11.0-17-generic #31~precise1-Ubuntu SMP Tue Feb 4
+21:25:43 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
+----
+
+==== Red Hat Enterprise Linux/CentOS Linux
+
+[IMPORTANT]
+====
+* If running the RHEL linux distribution, you must have a current valid
+  license from RedHat.
+* The server must have a valid yum or apt-get configuration so that it
+  can reach the Zimbra package servers.
+====
+
+* Red Hat® Enterprise Linux® 7, AS/ES (64-bit) 
+* CentOS Linux® 7 (64-bit)
+* Red Hat Enterprise Linux 6, AS/ES (64-bit), patch level 4 or later is required
+* CentOS Linux 6 (64-bit), patch level 4 or later is required
+
+=== License Activation
+
+[IMPORTANT]
+====
+* At the beginning of an upgrade installation, the existing license is
+  validated as being current and qualifies for the upgrade. If your
+  license is expired, an error message displays and the upgrade cannot
+  be performed. Contact {product-provider} Sales for a license renewal to continue
+  your upgrade.
+* An upgrade installation will not proceed without automatic
+  activation or a manually activated license file. License activations
+  are limited to five activations per license file. If you have
+  previously used all activations prior to upgrading your production
+  system, you must contact {product-provider} Sales to enable additional license
+  activations.
+====
+
+All {product-edition-commercial} installations require a valid license and license
+activation. New installs will have a 10 day grace period from the
+license issue date before requiring activation.
+
+License activation is automatic during the install with systems that
+have external access to the {product-provider} license servers. A means of
+creating manual activations will be provided for systems that do not
+have external access to the {product-provicder} license servers. See the
+_{product-admin-guide}_ for more information.
+
+
+When upgrading, the way in which ZCO and archiving licensing is
+enforced might have changed on the server if you are using an older
+version of {product-name}. Older licenses might have
+`MAPIConnectorAccountsLimit` set to `0` or `ArchivingAccountsLimit` missing
+in the license. Contact {product-provider} Sales for an updated license file prior
+to upgrading if you have licensed either of these features and your
+current license does not properly reflect the correct number.
+
+=== Upgrading LDAP Replica Servers or Multi-Master Server
+
+NOTE: These instructions apply to {product-abbrev} 8.0.0, 8.0.1, 8.0.2 to {product-abbrev}
+8.0.4 and later.
+
```

Comment:
The text
> If you have replica servers or are in multi-master mode, you have to install the Zimbra LDAP schema specific to the release you are upgrading to onto the replica servers or onto the multi-master server before you upgrade to ZCS 8.0.4 and later. (Bug 81048)
Has been lost from the original document which I think is useful.  Perhaps use an HTML link for the bug?